### ISSUE TITLE
How to add for='...' to Label?

### ISSUE THREAD

**Issue by BKFlister** [thomasborgen/hypermedia#34]
What is the correct/best method to add  `for='...'` to `Label`?

This will fail:

```python
Label('Username', for='username'),
Input(id='username', name='username', type='text', placeholder='Enter your username', required=True)
```

This is working:

```python
Label('Username', **{'for':'username'}),
Input(id='username', name='username', type='text', placeholder='Enter your username', required=True)
```

I see that FastHTML/FastTag uses `fr` to handle `for`

```python
Label('Username', fr='username'),
Input(id='username', name='username', type='text', placeholder='Enter your username', required=True)
```

FastHTML/FastTag  
https://www.fastht.ml

FastTag - Python library that mirrors fasttags from FastHTML/FastCore  
https://github.com/adamritter/fasttag

**Comment by thomasborgen** [comment]
I'm super stoked that you are actively trying out hypermedia and making issues.

`for_` should already work and be autocompleted for `Label` and `Output` elements. 

```python
Label(for_="username")
```

https://github.com/thomasborgen/hypermedia/blob/b291a047d8d26f23a199383eb5bace55f5b9aed4/hypermedia/types/attributes.py#L518

and 

https://github.com/thomasborgen/hypermedia/blob/b291a047d8d26f23a199383eb5bace55f5b9aed4/hypermedia/types/attributes.py#L363

I think all the attributes that conflicts with keywords has a trailing `_` underscore. :) ie: `class_`, but with `class_` you can use `classes=[]` instead.

**Comment by BKFlister** [comment]
Thanks,

```python
Label("Username", for_="username")
```

Results in a `for` with a hyphen, `for-` ?
```htm
<label for-="username">Username:</label>
```

`class_` result in `class`without a hyphen.

**Comment by thomasborgen** [comment]
Thanks for checking and Sorry... the Alias was not set up correctly.

**Comment by thomasborgen** [comment]
Fixed, will be released together with another issue. :)

**Comment by thomasborgen** [comment]
Fixed in `5.2.0` release :)
